tests/conftest.py

```diff
@@ -1,7 +1,28 @@
+import asyncio
+
 import pytest
+import aiohttp
 
 
 @pytest.fixture(scope="session", params=[True, False],
                 ids=['debug:true', 'debug:false'])
 def debug(request):
     return request.param
+
+
+@pytest.fixture
+def loop(event_loop, debug):
+    event_loop.set_debug(debug)
+    return event_loop
+
+
+@pytest.yield_fixture
```

Comment:
`yield_fixture` decorator is deprecated
http://doc.pytest.org/en/latest/yieldfixture.html